api/slack/transcription_check/messages.py

```diff
@@ -49,7 +49,7 @@ def update_check_message(check: TranscriptionCheck) -> None:
 def reply_to_action(data: Dict, text: str) -> Dict:
     """Reply to the given action with the given text."""
     channel_id = data["channel"]["id"]
-    message_ts = data["message_ts"]
+    message_ts = data.get("message", {}).get("ts")
```

Comment:
I'm not sure about this one but the rest looks :100:

Reply:
What are you not sure about?

Reply:
See https://api.slack.com/reference/interaction-payloads/block-actions#examples for the response we are getting

Reply:
Just that it's been a while since I worked with the contents of the data dict, so I don't really remember what is in the message section. Why swap to using the `ts` from message instead of the named variable?

Reply:
Ah! Thanks for the docs link. I see. Carry on 😄

Reply:
The `message_ts` property doesn't exist in the action payload, just in the response once you sent a message. Don't ask me why, I found it out the hard way by running into an index error

Reply:
Because slack is mean and wants to fuck up our day, that's why lol